tools/webpack/packages.js

```diff
@@ -152,6 +152,16 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 		to: join( baseDir, `src/${ phpFiles[ filename ] }` ),
 	} ) );
 
+	const phpMetadataCopies = Object.keys( phpFiles ).map( function( filename ) {
+		return {
+			filename: filename,
+			dir:dirname( filename ),
+		  }
+	} ).map( ({ filename, dir }) => ( {
+		from: join( baseDir, `node_modules/@wordpress/${ dir }/block.json` ),
+		to: join( baseDir, `src/${ dirname(phpFiles[ filename ]) }/${ basename(dir) }.json` ),
```

Comment:
Nit, spaces are missing:
```suggestion
		to: join( baseDir, `src/${ dirname( phpFiles[ filename ] ) }/${ basename( dir ) }.json` ),
```